sumologic/resource_sumologic_generic_polling_source.go

```diff
@@ -315,6 +343,10 @@ func resourceToGenericPollingSource(d *schema.ResourceData) (PollingSource, erro
 		URL:          d.Get("url").(string),
 	}
 
+	if source.ContentType == "AzureEventHubLog" {
+		pollingSource.ScanInterval = 300000
```

Comment:
are we setting it because this is a required field in TF? it's optional in Sumo so I wasn't expecting we have to set it explicitly

Reply:
We will encounter this error even we have already set the scan_interval as an optional field. 
<img width="1100" alt="Screenshot 2024-04-25 at 4 45 02 PM" src="https://github.com/SumoLogic/terraform-provider-sumologic/assets/124813412/7356e403-1968-4c26-9325-80d4cb555d75">

Reply:
That means it's sending 0 when creating the source, but I think TF default value for `scan_interval` should match that of Sumo API

Reply:
What do other sources do for scan_interval? I'm not a TF expert, why is an optional field giving a default value in TF?

Reply:
Linking slack thread: https://sumologic.slack.com/archives/CAU3N0Y92/p1714602126756289